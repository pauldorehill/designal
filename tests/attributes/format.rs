use designal::Designal;

#[derive(Designal)]
#[designal(rename = "NewBean", ,)]
struct HumanBean();

#[derive(Designal)]
#[designal(snozzcumber)]
struct HumanBean1();

fn main() {}