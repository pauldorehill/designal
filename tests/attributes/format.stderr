error: expected identifier or literal
 --> $DIR/format.rs:4:32
  |
4 | #[designal(rename = "NewBean", ,)]
  |                                ^

error: Unrecognized attribute identifier option: snozzcumber
 --> $DIR/format.rs:8:12
  |
8 | #[designal(snozzcumber)]
  |            ^^^^^^^^^^^
